perfkitbenchmarker/benchmarks/object_storage_service_benchmark.py

```diff
@@ -865,6 +878,24 @@ def Cleanup(self, vm):
     benchmark_spec_class.AZURE: AzureBlobStorageBenchmark()}
 
 
+WRONG_REGION_FLAGS = {
```

Comment:
It seems to be an unnecessary restriction to me. e.g., We allow testing from a GCE VM to S3 bucket, so it's actually a legitimate case here.